src/pdsfile/_preload.py

```
@@ -196,11 +196,13 @@ def get_permanent_values(cls, holdings_list, port):
                             continue
 
                         key = (pdsf1.logical_path + '/' + bundlename).lower()
-                        pdsf2 = cls.CACHE[key]
+                        # The value is not needed; the lookup is a presence
+                        # probe whose KeyError triggers the reload below.
+                        _ = cls.CACHE[key]
 
         except KeyError as e:
             cls.LOGGER.warn('Permanent value %s missing from Memcache; '
```

Ruff (0.16.0)

[warning] Logging statement uses `warn` instead of `warning`

Convert to `warning`

(G010)